app/org/sagebionetworks/bridge/services/SchedulePlanServiceImpl.java

```diff
@@ -59,23 +55,29 @@ public SchedulePlan getSchedulePlan(StudyIdentifier studyIdentifier, String guid
     }
 
     @Override
-    public SchedulePlan createSchedulePlan(SchedulePlan plan) {
+    public SchedulePlan createSchedulePlan(Study study, SchedulePlan plan) {
         checkNotNull(plan);
-        
+
+        // Plan must always be in user's study
+        plan.setStudyKey(study.getIdentifier());
+
```

Comment:
Might be worth adding a unit test to where the user tries to create a schedule in the wrong study, and we set the correct study.

From a semantics point of view, would it be better to silently set the correct study or throw a 400 Bad Request?